code-examples/quarkus-service-registry-kafka-streams-quickstart/src/main/java/org/acme/kafka/kstreams/OrdersTransformer.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.acme.kafka.kstreams;
+
+
+import org.acme.kafka.schema.avro.Event;
+import org.acme.kafka.schema.avro.Order;
+import org.apache.kafka.streams.KeyValue;
+import org.apache.kafka.streams.kstream.Transformer;
```

Comment:
Discussion on Kafka Sounding board suggested us to use Mutiny/Official Quarkus Kafka streams guide:
https://quarkus.io/guides/kafka-streams

Reply:
Ok, then I guess we can close this one. Let me know if you want/need help with the other one, I'm happy to help.